### DISCUSSION TITLE
[Feature] support for /e/os webdav accounts

### DISCUSSION THREAD

**Discussion by Derisis13** [TechbeeAT/jtxBoard#1766] [Q&A]
**Is your feature request related to a problem? Please describe.**

/e/os has a built-in webdav client. It's a fork of dav5x, the source is accessible here: https://gitlab.e.foundation/e/os/AccountManager
JTX board currently doesn't recognize this as a viable backend.

**Describe the solution you'd like**

I wish to use this built-in account manager/webdav client as a synchronization backend instead of separately installing dav5x.

**Describe alternatives you've considered**

Dav5x works fine for synchronization, as far as I can tell, it isn't bothered by the /e/ client.

**Additional context**

If work from the /e/ side is required to achieve this, leave it be, since I can't upgrade to the latest release due to a bug.

**Your current device and version**
 - Device: [Xiaomi Redmi Note 8T]
 - Android version: [Android 13]
 - jtx Board version: [2.09.01]
 - Downloaded from: [/e/'s "App Lounge", which I think gets it from F-droid but I'm not 100% on that]

**Comment by BaumiCoder** [comment]
The reason for this problems seems to be a change in DAVx5 fork of the /e/ project, as @patrickunterwegs explaint in a [GitHub Dicussion](https://github.com/TechbeeAT/jtxBoard/discussions/120). An issue at /e/ project about this exists [here](https://gitlab.e.foundation/e/backlog/-/issues/6508), but was closed as inactive.

**Comment by Derisis13** [comment]
Oh I see. Sorry I made a duplicate, I didn't look at the discussions, only issues.
I find it very unlikely that /e/ will do anything about their fork, it  hasn't seen any activities for years from what I've seen. 
Feel free to close the issue if there's nothing you can do with reasonable effort.

**Comment by patrickunterwegs** [comment]
Hi, sorry for the late reply on this. Life has been a bit harsh in the last couple of months. However, trying to catch up what I had to leave behind. 
Yes, as pointed out by @BaumiCoder this is a thing /e/ would have to do, see also https://gitlab.e.foundation/e/backlog/-/issues/6508

I will still move this topic to discussions to keep it open for other users.
